### ISSUE TITLE
Changing justTouchEvents using window.tocca doesn't do anything

### ISSUE THREAD

**Issue by cpitclaudel** [GianlucaGuarini/Tocca.js#72]
As far as I can tell, the justTouchEvents variables is read from defaults just once, and changing it through `window.tocca()` doesn't do anything.  Am I missing something?

**Comment by GianlucaGuarini** [comment]
that's right and it's meant to work in this way by design to keep the Tocca.js code as simple as possible. I don't see any reason to add/remove event listeners multiple times once the script has been loaded.

**Comment by cpitclaudel** [comment]
Then the docs need to be updated:

```javascript
window.tocca({
  useJquery: your new option
  swipeThreshold: your new option
  tapThreshold: your new option
  dbltapThreshold: your new option
  longtapThreshold: your new option
  tapPrecision: your new option
  justTouchEvents: your new option // don't use
})
```

But maybe I didn't explain the issue well.  Since Tocca.js adds listeners as soon as it is loaded, there's no way to turn of touch events using the `window.tocca` function — one must use the `JUST_ON_TOUCH_DEVICES` global.  Am I missing something?

**Comment by GianlucaGuarini** [comment]
> Then the docs need to be updated:

Fair enough. Can you please prepare a PR for it? thanks

**Comment by cpitclaudel** [comment]
I can, but I don't get it it: what's the point of that argument if it's not used?  Should it just be removed entirely?
